src/index.js

```diff
@@ -24,16 +24,19 @@ const transform = (successFn: Function, failFn: Function, input: Array<any>): an
 /**
  *
  * @param {Function} predicate validation function to apply inputs on
- * @param {String} errorMsg error message to return in case of fail
+ * @param {String|Function} errorMsg error message to return in case of fail
  * @param {*} value the actual value
  * @param {Object} inputs the input object - in case the predicate function needs access to dependent values
  * @returns {Boolean}
  */
 const runPredicate = ([predicate, errorMsg]:[Function, string],
   value:any,
-  inputs:Object) => predicate(value, inputs)
-  ? true
-  : errorMsg
+  inputs:Object) => {
+  if (predicate(value, inputs)) {
```

Comment:
@busypeoples Sure? That's how I wrote it originally.

Did you mean disabling the rule in the config or just here with a `eslint-disable-line`?